app/models/upload.rb

```diff
@@ -10,6 +10,7 @@ class Upload < ActiveRecord::Base
   SEEDED_ID_THRESHOLD = 0
   URL_REGEX ||= /(\/original\/\dX[\/\.\w]*\/(\h+)[\.\w]*)/
   MAX_IDENTIFY_SECONDS = 5
+  MAX_DOMINANT_COLOR_SECONDS = 5
```

Comment:
```suggestion
  DOMINANT_COLOR_COMMAND_TIMEOUT_SECONDS = 5
```

Small suggestion as the previous constant name didn't really reflect what the constant was going to be used for.